images/installer/nested-libvirt/README.md

```diff
@@ -0,0 +1,41 @@
+# Nested libvirt on GCE
+
+This image enables launching a libvirt cluster nested in a GCE instance, for libvirt CI testing.
+
+This image contains [nss_wrapper](https://cwrap.org/nss_wrapper.html) to execute ssh commands as 
+a mock user to interact with a GCE instance from an OpenShift container.
+
+OpenShift containers run with an arbitrary uid, but ssh requires a valid user.  `nss_wrapper`
+allows for the container's uid to be mapped to a user inside of a container.
+
+### Example Usage
+
+You can override the container's current uid and gid by providing NSS_USERNAME, NSS_UID, NSS_GROUPNAME, and/or NSS_GID. In OpenShift CI, we only use NSS_USERNAME and NSS_GROUPNAME.
+
+```
+$ docker run --rm \
+-e NSS_USERNAME=packer \
+-e NSS_GROUPNAME=packer \
+nss_wrapper-image -c '/bin/generate-user.sh && id packer'
```

Comment:
You shouldn't need `/bin/` here, because `/bin` should be in the container's `PATH`:

```console
$ podman run --rm -it docker.io/library/centos:7 env | grep PATH
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Reply:
fixed